types/react-select/src/Async.d.ts

```diff
@@ -18,7 +18,7 @@ export interface AsyncProps<
      * Function that returns a promise, which is the set of options to be used
      * once the promise resolves.
      */
-    loadOptions: (
+    loadOptions?: (
```

Comment:
Isn't there a combination of props where this method is required?

Reply:
From a practical sense, yes, but from a type-safety sense it's not required (i.e., everything will work as designed even if no props are provided to this component, there will just never be any options in the Select dropdown).